### PR TITLE
Add Controller Admin Guide

### PR DESCRIPTION
**Pull request by EMcWhinn**
Add title for Controller Admin started guide

Add master.adoc and docinfo.xml and symlinks

Use assemblies and modules from /platform directories

https://issues.redhat.com/browse/AAP-17254

Affects /titles/controller/controller-getting-started/

### CODE REVIEW

File: downstream/titles/controller/controller-admin-guide/master.adoc [hunk 1]
```diff
@@ -0,0 +1,14 @@
+:imagesdir: images
+:numbered:
+:toclevels: 1
+
+:experimental:
+:controller-UG:
```
Comment (ianf77) [inline downstream/titles/controller/controller-admin-guide/master.adoc:6]: :controller-AG:  or :controller-Admin:

File: downstream/titles/controller/controller-admin-guide/master.adoc [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+include::attributes/attributes.adoc[]
+
+
+// Book Title
+= Automation controller Administration Guide
```
Comment (ianf77) [inline downstream/titles/controller/controller-admin-guide/master.adoc:12]: Automation Controller

**Review by ianf77 — COMMENTED** [review]
Couple of comments

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**